mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping

```diff
@@ -0,0 +1,83 @@
+CLASS net/minecraft/unmapped/C_rcqaryar net/minecraft/entity/ai/brain/task/TaskBuilder
+	FIELD f_fbolpvvc trigger Lnet/minecraft/unmapped/C_rcqaryar$C_gcagkauk;
+	METHOD m_baiqplax unbox (Lcom/mojang/datafixers/kinds/App;)Lnet/minecraft/unmapped/C_rcqaryar;
+		ARG 0 app
+	METHOD m_bqiptaml (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;)Lcom/mojang/datafixers/kinds/App;
+		ARG 1 instance
+	METHOD m_cjmctyax (Ljava/util/function/BiPredicate;Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_usxaxydn;J)Z
+		ARG 2 entity
+		ARG 3 time
+	METHOD m_ckmuzzhw (Lnet/minecraft/unmapped/C_smovfozc;Lnet/minecraft/unmapped/C_smovfozc;Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;)Lcom/mojang/datafixers/kinds/App;
+		ARG 2 instance
+	METHOD m_dtlemngu getFunction (Lcom/mojang/datafixers/kinds/App;)Lnet/minecraft/unmapped/C_rcqaryar$C_gcagkauk;
+	METHOD m_elxiakeb task (Ljava/util/function/Function;)Lnet/minecraft/unmapped/C_yfpegcpm;
+		ARG 0 creator
+	METHOD m_fbtxuyrh (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_usxaxydn;J)Z
+		ARG 2 entity
+		ARG 3 time
+	METHOD m_fjlwqoug (Ljava/util/function/BiPredicate;Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;)Lcom/mojang/datafixers/kinds/App;
+		ARG 1 instance
+	METHOD m_hfypbtrs instance ()Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;
+	METHOD m_ipranrme sequence (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_yfpegcpm;)Lnet/minecraft/unmapped/C_yfpegcpm;
+		ARG 0 predicate
+		ARG 1 task
+	METHOD m_niacdizy triggerIf (Ljava/util/function/Predicate;)Lnet/minecraft/unmapped/C_yfpegcpm;
+		ARG 0 predicate
+	METHOD m_uxcmzqzq triggerIf (Ljava/util/function/BiPredicate;)Lnet/minecraft/unmapped/C_yfpegcpm;
+		ARG 0 predicate
+	METHOD m_vpobqcer create (Lnet/minecraft/unmapped/C_rcqaryar$C_gcagkauk;)Lnet/minecraft/unmapped/C_rcqaryar;
+		ARG 0 trigger
+	METHOD m_xtjbildx sequence (Lnet/minecraft/unmapped/C_smovfozc;Lnet/minecraft/unmapped/C_smovfozc;)Lnet/minecraft/unmapped/C_yfpegcpm;
+		ARG 0 predicate
+		ARG 1 task
+	CLASS C_gcagkauk TriggerWithResult
+		METHOD m_dlmxufff run (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_usxaxydn;J)Ljava/lang/Object;
+			ARG 2 entity
+			ARG 3 time
+		METHOD m_ioeqqkda toDebugString ()Ljava/lang/String;
+	CLASS C_hzjatjhv Constant
+		METHOD <init> (Ljava/lang/Object;)V
+			ARG 1 value
+		METHOD <init> (Ljava/lang/Object;Ljava/util/function/Supplier;)V
+			ARG 1 value
+			ARG 2 debugStringSupplier
+	CLASS C_loqvwuht Instance
+		METHOD ap2 (Lcom/mojang/datafixers/kinds/App;Lcom/mojang/datafixers/kinds/App;Lcom/mojang/datafixers/kinds/App;)Lcom/mojang/datafixers/kinds/App;
```

Comment:
```suggestion
		METHOD app2 (Lcom/mojang/datafixers/kinds/App;Lcom/mojang/datafixers/kinds/App;Lcom/mojang/datafixers/kinds/App;)Lcom/mojang/datafixers/kinds/App;
```
?

Reply:
I must have marked an automapped name as deobf, that name is included in the jar for some reason